kmod/patch/kpatch-syscall.h

```diff
@@ -91,21 +91,20 @@
 
 #elif defined(CONFIG_S390)
 
-# if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 3, 0)
-#  define KPATCH_SYSCALL_WRAPPERS_V2
-# else
-#  define KPATCH_SYSCALL_WRAPPERS_V1
-# endif
-
 # if defined(RHEL_RELEASE_CODE)
 #  if RHEL_RELEASE_CODE >= RHEL_RELEASE_VERSION(9, 6)
 #   define KPATCH_SYSCALL_WRAPPERS_V2
 #  else
 #   define KPATCH_SYSCALL_WRAPPERS_V1
 #  endif
+# else
+#  if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 3, 0)
+#   define KPATCH_SYSCALL_WRAPPERS_V2
+#  else
+#   define KPATCH_SYSCALL_WRAPPERS_V1
+#  endif
 # endif
 
```

Comment:
@jpoimboe : how did we miss this in the review :laughing: ?  I think I was so annoyed about not being able to combine `defined(X) ... X` on a single line that I botched the rest.